docs/components/page-title.md

```diff
@@ -17,6 +17,9 @@ eleventyNavigation:
 - Pelican recommends a Page Title for every page.
 - Use the button to provide quick access to one common action for the page.
 - If a page needs more than one button, use a [Bootstrap Dropdown button](https://getbootstrap.com/docs/5.2/components/dropdowns/#single-button).
+- Remove the [Badges](/components/badges/) section if you don’t need them.
+- Try to keep to just two or three Badges.
+- Badges in the Page Title will appear dimmed unless they are Danger or Warning Badges.
```

Comment:
Does this also imply that chips will have this same effect too?

Reply:
@frogpad7